src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,26 @@
-export const Tabs = () => {};
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }) => (
+  <div className="tabs is-boxed">
+    <ul>
+      {tabs.map(tab => (
+        <li
+          key={tab.id}
+          className={selectedTabId === tab.id ? 'is-active' : ''}
```

Comment:
looks like here the reason of fail one of test.
Find selected tab by selectedTabId or assign tabs[0] and here compare with selectedTab
```
const selectedTab  = tabs.find(tab => tab.id === selectedTabId) || tabs[0]
```
```suggestion
          className={selectedTab.id === tab.id ? 'is-active' : ''}
```